config/_default/config.toml

```diff
@@ -43,6 +43,7 @@ home = [ "HTML", "RSS", "JSON"]
   # CRS related parameters
   crs_latest_release = "3.3.4"
```

Comment:
This better be 4.0.0, I guess.

Reply:
The build failed to I've reverted this change. Will push it after release.